lib/utils/sdk.ts

```diff
@@ -47,3 +48,27 @@ export async function getApplication(
 	}
 	return sdk.models.application.get(nameOrSlugOrId, options);
 }
+
+/**
+ * Wraps the sdk organization.getAll method,
+ * restricting to those orgs user is a member of
+ */
+export async function getOwnOrganizations(
+	sdk: BalenaSDK,
+): Promise<Organization[]> {
+	return await sdk.models.organization.getAll({
+		$filter: {
```

Comment:
We may or may not want some additional filtering here, because I got the following `BalenaRequestError` while testing the PR:
```
$ ./bin/balena whoami
== ACCOUNT INFORMATION
USERNAME: gh_paulo_castro
EMAIL:    pefcastro@gmail.com
URL:      balena-cloud.com

$ ./bin/balena app create test33 --type raspberrypi3
? Select an organization (Use arrow keys)
❯ gh_paulo_castro’s Organization
  g_paulo_castro1’s Organization
  Page’s Organization
  unicorn’s Organization

? Select an organization g_paulo_castro1’s Organization
BalenaRequestError: Request error: Unauthorized
```
I think what happened is that user `gh_paulo_castro` (the user logged in to the CLI) had been added to another user's org, `g_paulo_castro1’s Organization` that belongs to `g_paulo_castro1`, with the "member" role that the web UI describes as "ready-only access to the org", thus lacking authorization to create apps in that org.

I think that the nicest UX for `balena app create` would be to filter out orgs where the app cannot be created. The second best (also acceptable) would be to issue an error, which actually means doing nothing because the code already does that, and apparently treating the error as 'expected', which is good.

Reply:
@thgreasi Could you add your opinion here please?

Reply:
@pdcastro Right, I didn't know about that.   I suppose one point of view is that it's better to keep all associated orgs on the list, and then throw a permissions error if they try to use a read only one (maybe a more friendly error).  At least that way they will know why it doesn't work, rather than an org juts missing from the list, which might cause more support queries.

Reply:
@pdcastro If you have time, could you try adding this console log, and posting the data for `g_paulo_castro1’s Organization`
```
	async getOrganization() {
		const { getOwnOrganizations } = await import('../../utils/sdk');
		const organizations = await getOwnOrganizations(getBalenaSdk());

		console.log(organizations);
```

Reply:
@pdcastro Please take a look at the revised version I just pushed, which handles that error.  I think I prefer this solution as it keeps everything transparent.

Reply:
> could you try adding this console log, and posting the data for `g_paulo_castro1’s Organization`?

@srlowe 
```
  {
    id: 30460,
    created_at: '2018-10-18T20:38:20.946Z',
    name: 'g_paulo_castro1’s Organization',
    handle: 'g_paulo_castro1',
    company_name: 'Balena',
    website: null,
    industry: null,
    billing_account_code: null,
    __metadata: { uri: '/resin/organization(@id)?@id=30460' }
  }
```

Reply:
@srlowe if you do add a filter on organization_member_role it would stop working on oB since we don't have that there.
More info in my other comment (I didn't see this before)
See: https://github.com/balena-io/balena-cli/pull/2127#pullrequestreview-551444748